examples/c/ex_all.c

```diff
@@ -900,6 +900,23 @@ transaction_ops(WT_SESSION *session_arg)
 	error_check(session->commit_transaction(session, NULL));
 	/*! [transaction isolation] */
 
+	/*! [transaction prepare] */
+	/*
+	 * An active transaction can be prepared, if successful will guarantee
+	 * a commit. No operations, apart from commit/rollback operation is
+	 * allowed on a prepared transaction, including session/cursor
+	 * operations. At the moment, prepare transaction is designed to
+	 * support mongodb exclusively.
+	 */
+	error_check(session->open_cursor(
+	    session, "table:mytable", NULL, NULL, &cursor));
+	error_check(session->begin_transaction(session, NULL));
+	cursor->set_key(cursor, "key");
+	cursor->set_value(cursor, "value");
+	session->prepare_transaction(session, "prepare_timestamp=2a");
```

Comment:
I think you need to error check the `prepare_transaction` call.

Reply:
If error check is done, build will fail as currently prepare_transaction will return error. Will update this example as part of wt-3848.